styles/InfoBlock.module.css

```diff
@@ -0,0 +1,76 @@
+.info_block {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  gap: 48px;
+  padding: 64px 0;
+  border-bottom: 1px solid var(--border-default);
+}
+
+.info_block__title {
+  grid-column: 1/3;
+}
+
+.info_block_heading {
+  text-transform: uppercase;
+}
+
+.info_block__title_text {
+  font-weight: 500;
+  text-transform: uppercase;
+  color: var(--text-default);
+  margin: 16px 48px 16px 0;
+  height: 424px;
+}
+
+.info_block__icon {
+  color: var(--icon-default);
+}
+
+.info_block__normal_content {
+  display: flex;
+  flex-direction: column;
+  align-content: space-between;
+  grid-column: 3;
+  height: 571px;
+  padding-left: 48px;
+  border-left: 1px solid var(--border-default);
+}
```

Comment:
I want to ask that if it is single line, wether it will be better to use flex?

Reply:
This grid divided the width of its container into equal 3 columns even we just have 2 grid items.